app/src/components/prettyPageWrap/PrettyPageWrap.tsx

```diff
@@ -46,7 +54,10 @@ const PrettyPageWrap: React.FunctionComponent = ({
                         </>
                     )}
                     <SignOutButton setLoggedIn={setLoggedIn} />
-                    <NavItem>{getCurrentUserName()}</NavItem>
+                    <NavItem>{getCurrentUserName().split(" ")[0]}</NavItem>
```

Comment:
No, can we use the full name please.
Splitting on a space does make some assumptions about names that won't always hold true.